src/packages/excalidraw/CHANGELOG.MD

```diff
@@ -4,6 +4,17 @@
 
 First release of `@excalidraw/excalidraw`
 
+### [Unreleased]
+Aligns arrowhead schemas [#2517](https://github.com/excalidraw/excalidraw/pull/2517)
+Expand canvas padding based on zoom. [#2515](https://github.com/excalidraw/excalidraw/pull/2515)
+Fix Library Menu Layout [#2502](https://github.com/excalidraw/excalidraw/pull/2502)
+Add Cut to menus [#2511)](https://github.com/excalidraw/excalidraw/pull/2498)
+hide stats and scrollToContent-button when mobile menus open [#2509](https://github.com/excalidraw/excalidraw/pull/2509)
+Hide shortcuts on pickers for mobile [#2508](https://github.com/excalidraw/excalidraw/pull/2508)
+More Arrowheads: dot, bar [#2486](https://github.com/excalidraw/excalidraw/pull/2486)
+Bump ini from 1.3.5 to 1.3.7 in /src/packages/excalidraw [#2500](https://github.com/excalidraw/excalidraw/pull/2500)
+Support CSV graphs and improve the look and feel [#2495](https://github.com/excalidraw/excalidraw/pull/2495)
+
```

Comment:
Make it as a list.. it's not good in the [preview](https://github.com/excalidraw/excalidraw/blob/aakansha-changelog/src/packages/excalidraw/CHANGELOG.MD)

Reply:
My bad. fixed